classes/music_album.rb

```diff
@@ -0,0 +1,17 @@
+require 'date'
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, publish_date, archived)
+    super(publish_date, archived)
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archived
+    return true if super && @on_spotify
+  end
```

Comment:
should return false in case the return is not true
just add false before the end keyword

Reply:
Hi @tresorsawasawa 
The method should look like this
```
def can_be_archived
    return true if super && @on_spotify

    false
  end
```